.rubocop.yml

```diff
@@ -13,8 +13,10 @@ AllCops:
     - 'docs/api/restility/**/*'
     - 'src/api/test/fixtures/backend/**/*'
     - 'src/api/files/*'
-    - 'dist/**/*'
     - 'src/backend/**/*'
+    - 'dist/obs-api-deps.spec'
```

Comment:
I'll add a comment for the reason to exclude them (as you did in your commit) :wink: